### DISCUSSION TITLE
Add pagination to view older feeds

### DISCUSSION THREAD

**Discussion by see2ever** [osmoscraft/osmosfeed#35] [Ideas]
when the feeds grow large, it's hard to scroll down to the bottom

**Comment by chuanqisun** [comment]
Hey @see2ever, I have several ideas in mind, I'm curious if any might address your issue:
- A simple keyboard shortcut to scroll to bottom (try holding <kbd>space</kbd>)
- A sticky sidebar that allows to you jump to a week or month
- Auto restore scroll position when you navigate away and come back to the site.

I can also consider the classic pagination at the bottom, with first, last, prev, next, and some page number in the middle. Could you provide more context? e.g. What are you trying to achieve by scrolling to the bottom? Usually, people find pagination slower to use than long scroll but I'm sure you have specific reasons. I just want to make sure I understand your needs before adding a new feature.

Thanks!

**Reply by see2ever** [reply]
just to check every item had been read, with page help, i can navigate to pages which hasn't been read.

Thanks!
